Android/app/src/main/java/com/example/kahye/common/ReservationActivity.java

```diff
@@ -49,39 +58,53 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_reservation);
 
         // class Img
-        //Todo(woongjin) change the hardcoded url to read config file and use it
-        String imageURL = "http:10.0.2.2:8000" + bundle.getString("classImgURL");
+        // Todo(woongjin) change the hardcoded url to read config file
+        String imageURL = "http://52.8.187.167:8000" + bundle.getString(
+                "classImgURL");
         ImageView classImgView = (ImageView) findViewById(R.id.classImgView);
         Picasso.get().load(imageURL)
                 .fit()
                 .into(classImgView);
 
         // class Info
-        TextView classInfoView = (TextView)findViewById(R.id.classInfoView);
-        String className = bundle.getString("className");
-        classInfoView.setText("Class: " + selectedClass.getClassName() + "\nExpert: "
-                + selectedClass.getExpertName() + "\nMin: "
-                + selectedClass.getMinGuestCount().toString() +  "\nMax: "
-                + selectedClass.getMaxGuestCount().toString());
+        classNameView = (TextView)findViewById(R.id.classTextView);
+        expertNameView = (TextView)findViewById(R.id.expertTextView);
+        numOfPeopleView = (TextView) findViewById(R.id.numOfPeopleView);
+        priceView = (TextView) findViewById(R.id.priceView);
+
+        className = bundle.getString("className");
+        classNameView.setText(selectedClass.getClassName());
+        expertNameView.setText(selectedClass.getExpertName());
+        numOfPeopleView.setText(selectedClass.getMinGuestCount().toString()
+                + " - " + selectedClass.getMaxGuestCount().toString());
+        priceView.setText(selectedClass.getPrice().toString());
+
+        classNameView.setBackgroundColor(Color.parseColor(
+                "#9931343a"));
+        expertNameView.setBackgroundColor(Color.parseColor(
+                "#9931343a"));
 
         selectedDate = bundle.getString("_date");
         TextView dateView = (TextView) findViewById(R.id.dateView);
         dateView.setText(selectedDate);
 
         // time list
-        final ListView timeListView = (ListView) findViewById(R.id.timeListView);
+        final ListView timeListView = (ListView) findViewById(
+                R.id.timeListView);
         final List<String> timeList = new ArrayList<>();
         ArrayAdapter<String> adapter = new ArrayAdapter<>(this,
                 android.R.layout.simple_list_item_1, timeList);
 
         List<TimeTable>  timeslot = selectedClass.getAvailableTimeTable();
         for (int timeListIdx = 0; timeListIdx < timeslot.size(); timeListIdx++){
-            String timeString = timeslot.get(timeListIdx).getStartTime().toString() + " ~ " +
-                    timeslot.get(timeListIdx).getEndTime().toString();
+            String timeString = timeslot.get(timeListIdx).getStartTime().
+                    toString() + " ~ " + timeslot.get(timeListIdx).getEndTime
+                    ().toString();
```

Comment:
```
            String timeString = 
                    timeslot.get(timeListIdx).getStartTime().toString() + " ~ " 
                    + timeslot.get(timeListIdx).getEndTime().toString();
```
How about change the code like this?
Two codes are same but the above one is easy to read

Reply:
ok. I will change that code :)